alpine-ruby/Dockerfile

```diff
@@ -29,9 +29,9 @@ RUN echo 'gem: --no-document' > ~/.gemrc \
 
 # Setup bundler for the root user
 RUN bundle config --global -- build.nokogiri '--use-system-libraries' \
-  && bundle config --global -- frozen 'true' \
+  && bundle config --global -- frozen '1' \
```

Comment:
Interesting that `true` was not accepted and you had to change it to `1` but it did NOT error out.